.github/workflows/trivy.yml

```diff
@@ -0,0 +1,40 @@
+name: dsp-appsec-trivy
+on: [pull_request]
+
+jobs:
```

Comment:
Is it possible to configure this file so that we only see 1 github action that runs all these checks with an all or nothing pass/fail? The test section is already very long

Reply:
I think the idea of a separate action was to not break your builds! But given that we've used this tool throughput many repos for several months now, this doesn't happen often. So may as well just add it as part of your overall workflow.